qutebrowser/misc/guiprocess.py

```diff
@@ -125,6 +128,8 @@ def _on_finished(self, code, status):
                 log.procs.error("Process stderr:\n" + stderr.strip())
 
         qutescheme.spawn_output = self._spawn_format(exitinfo, stdout, stderr)
+        self.final_stdout = stdout
+        self.final_stderr = stderr
```

Comment:
This probably makes sense to have even if it's not used at the moment. Please add a `yield qutebrowser.misc.guiprocess.GUIProcess.final_stderr` to the whitelist in `scripts/dev/run_vulture.py` (probably under "Not used yet, but soon").